### PR TITLE
Dashboard ImagePart, SeparatorPart and CustomColor

### PR DESCRIPTION
**Pull request by caltane**

### CODE REVIEW

File: Signum.Engine.Extensions/Dashboard/DashboardLogic.cs [hunk 1]
```diff
@@ -60,11 +60,13 @@ public static void Start(SchemaBuilder sb, IFileTypeAlgorithm cachedQueryAlgorit
                 {"LinkListPart", typeof(LinkListPartEntity)},
                 {"ValueUserQueryListPart", typeof(ValueUserQueryListPartEntity)},
                 {"UserTreePart", typeof(UserTreePartEntity)},
+                {"ImagePart", typeof(UserTreePartEntity)},
```
Comment (olmobrutall) [inline Signum.Engine.Extensions/Dashboard/DashboardLogic.cs:63]: UserTreePartEntity?

File: Signum.Entities.Extensions/Dashboard/DashboardEntity.cs [hunk 2]
```diff
@@ -264,4 +266,18 @@ public enum DashboardEmbedededInEntity
     Tab
 }
 
+[InTypeScript(true)]
+public enum PanelStyle
```
Comment (olmobrutall) [inline Signum.Entities.Extensions/Dashboard/DashboardEntity.cs:270]: Remove PanelStyle and add only string? CustomColor, keep BootstrapStyle for Workflow

File: Signum.React.Extensions/Dashboard/Admin/SeparatorPart.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { TypeContext, ValueLine } from '../../../../Framework/Signum.React/Scripts/Lines';
+import { SeparatorPartEntity } from '../Signum.Entities.Dashboard';
+
+export default function SeparatorPart(p: { ctx: TypeContext<SeparatorPartEntity> }) {
+  const ctx = p.ctx.subCtx({ formGroupStyle: "SrOnly", placeholderLabels: true });
+
+  return (
+    <div className="form-inline">
+      <ValueLine ctx={ctx.subCtx(c => c.title)} />
```
Comment (olmobrutall) [inline Signum.React.Extensions/Dashboard/Admin/SeparatorPart.tsx:10]: Maybe add also Description for help text?

**Comment by olmobrutall** [comment]
Thanks for the improvements in Dashboard @caltane!

**Comment by MehdyKarimpour** [comment]
Great job👌

**Pull request merged**